repos/system_upgrade/el7toel8/actors/systemfacts/libraries/systemfacts.py

```diff
@@ -5,10 +5,11 @@
 import os
 import pwd
 import re
-import six
 import subprocess
 import logging
 from subprocess import CalledProcessError
+from leapp.libraries.stdlib import api
+import six
```

Comment:
I made this adjustment because of 
pylint systemfacts.py | grep six
No config file found, using default configuration
C:  9, 0: standard import "import subprocess" should be placed before "import six" (wrong-import-order)
C: 10, 0: standard import "import logging" should be placed before "import six" (wrong-import-order)
C: 11, 0: standard import "from subprocess import CalledProcessError" should be placed before "import six" (wrong-import-order)